packs/feat-effects/effect-crimson-shroud.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Crimson Shroud",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud].</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud].</p>\n<p>You gain fast healing equal to half your level. You gain the @UUID[Compendium.pf2e.actionspf2e.Item.Swirl Crimson Shroud] action.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud]</p>\n<p>You gain fast healing equal to half your level. You gain the @UUID[Compendium.pf2e.actionspf2e.Item.Swirl Crimson Shroud] action.</p>"
```

Reply:
Consider removing empty `"details"`  from RE.

Reply:
??
You mean empty space in the formula? that should be fixed now.